### ISSUE TITLE
Compiler Bisector Improvements

### ISSUE THREAD

**Issue by eellison** [pytorch/pytorch#143453]
### 🚀 The feature, motivation and pitch

@ezyang has been using Compiler Bisector internally and run it into a few feature requests. 

- [ ] Query for backend, subsystems
- [ ] Config option to check meta stride for all ops, not just custom ops
- [ ] Option to specify particular backend/subsystem to iterate over
- [ ] Better print outs of how to manually run a particular bisect - for instance, if we bisected lowerings it should inform user to compare: `TORCH_BISECT_SUBSYSTEM=lowerings TORCH_BISECT_BACKEND=inductor TORCH_BISECT_MAX=21` and `TORCH_BISECT_SUBSYSTEM=lowerings TORCH_BISECT_BACKEND=inductor TORCH_BISECT_MAX=20` 

Other requests 
- [ ] Option to bisect which compiled graph is causing the issue first. Potentially we would bisect to the bad fwd/backward. then see if fwd, back, or joint graph passes/partitioner is the issue.

cc @exclamaforte who was interested

**Comment by ezyang** [comment]
The bisect_status.txt persistent state should NOT affect code that is not instrumented with bisector. Right now, if you have /tmp/torchinductor_ezyang/bisect/bisect_status.txt set, it will silently cause REGULAR torch.compile calls EVERYWHERE to only run at the bisect stage here. REALLY REALLY confusing 😠

**Comment by eellison** [comment]
@ezyang how are you doing this ? it should be cleaned up on each run: https://github.com/pytorch/pytorch/blob/8d4926e30a944320adf434016129cb6788eff79b/torch/_inductor/compiler_bisector.py#L490-L495

**Comment by exclamaforte** [comment]
Could be run if someone runs Ctrl-C/SIGINT maybe? 
```
import signal
import sys
def txtcleanup(sig, frame):
    # run .txt cleanup
    pass
# Register the signal handler
signal.signal(signal.SIGINT, signal_handler)
```

**Comment by exclamaforte** [comment]
There's a bunch of ways a program could not finish tho, so maybe some sort of hashid to make sure that we're in the correct session

**Comment by ezyang** [comment]
Yeah not only am I ctrl-c but I am also running multiple programs concurrently on separate gpus
